### DISCUSSION TITLE
Make amount of first place ranks influence your global rank

### DISCUSSION THREAD

**Discussion by Doxxz** [ppy/osu#22029] [Ideas]
If you can set a #1 score in a beatmap, you have proven yourself to be better than anyone else attempting. At least for that map. I think that should show on your rank.

Maybe a small pp bonus for every first place rank? Of course, it should scale with star rating, so higher star maps rightfully reward more than lower star ones.

If more than one player at the top of the list gets the same score, they could all receive the bonus, or maybe no one gets anything.

If the future multiple lazer leaderboards is a concern, maybe only the "pp leaderboard" (stable Global Ranking) should count. If that's going to be a thing.

I think it would create some interesting PvP interactions, with players stealing pp from others by sniping first place. It would encourage improving old scores and maybe playing more maps too, to grind for more number ones.

**Comment by ghost** [comment]
isn't this basically ppv1

**Reply by Doxxz** [reply]
Maybe a bit, but I'm not suggesting for it to replace current pp, it would just be a bonus on top of the current system to reward good players setting sick scores that are above every other score.

**Comment by Rekunan** [comment]
This goes back to why bonus pp is bad, which is that the goal of the pp system is to become an objective metric of performance. To avoid repeating myself from a previous discussion, said goal becomes diminished if you add features with the intent of encouraging anything, rather than continuing further on the path to objectivity.

**Reply by mk56-spn** [reply]
while this is correct this doesnt take away the fact that PPV2 is still awful at this due to the fact that it weights decade old scores the same as new ones, while it is highly likely many of those plays form a decade ago could not be done by the players now

**Reply by Rekunan** [reply]
I don't think adding some sort of pp decay to inactive players is a good idea, because it's not meant to measure skill, but performance. When they set the play, its performance was worth x pp, and will be worth x pp forever (ignoring pp reworks) because it's still an x pp amount of performance.
